<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoSplatoon</class>
 <widget class="QMainWindow" name="AutoSplatoon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>325</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AutoSplatoon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/Splatoon_2_-_Green_Inkling_Squid_icon.png</normaloff>:/icons/Splatoon_2_-_Green_Inkling_Squid_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>335</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang TC</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="serialConnectButton">
           <property name="font">
            <font>
             <weight>50</weight>
             <family>PingFang TC</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>連接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="serialRefreshButton">
           <property name="font">
            <font>
             <weight>50</weight>
             <family>PingFang TC</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>50</weight>
             <family>PingFang TC</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>串口狀態：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="serialStatusLabel">
           <property name="font">
            <font>
             <weight>50</weight>
             <family>PingFang TC</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>斷連</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="serialPortsBox">
         <property name="font">
          <font>
           <family>PingFang TC</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="flashButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <family>PingFang TC</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>燒錄</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="manualButton">
           <property name="font">
            <font>
             <weight>50</weight>
             <family>PingFang TC</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>手動控制</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>330</width>
         <height>130</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>PingFang TC</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>模擬按鍵操作時間間隔（ms）</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>開始行數</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rowBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>119</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>開始列數</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="columnBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>319</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="uploadButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <family>PingFang TC</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>上傳BMP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>PingFang TC</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>開始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>PingFang TC</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>暫停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="haltButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>PingFang TC</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>結束</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
